experiments/templatetags/experiments.py

```diff
@@ -1,30 +1,53 @@
+# coding=utf-8
 from __future__ import absolute_import
 
+from operator import attrgetter
+from uuid import uuid4
+
 from django import template
 from django.core.urlresolvers import reverse
+from jinja2 import (
+    ext,
+    nodes,
+    TemplateSyntaxError,
+)
 
 from experiments.utils import participant
 from experiments.manager import experiment_manager
 from experiments import conf
 
-from uuid import uuid4
 
 register = template.Library()
 
 
 @register.inclusion_tag('experiments/goal.html')
 def experiment_goal(goal_name):
-    return {'url': reverse('experiment_goal', kwargs={'goal_name': goal_name, 'cache_buster': uuid4()})}
+    return _experiment_goal(goal_name)
+
+
```

Comment:
extra lines? I think we add 2 lines to seperate classes right?

Reply:
By PEP8, things at the root of a module should be separated by 2 lines. Linter required it.